src/static/riot/queues/management.tag

```diff
@@ -1,15 +1,29 @@
 <queues-list>
-    <div class="ui icon input">
-        <input type="text" placeholder="Search by name..." ref="search" onkeyup="{filter.bind(this, undefined)}">
-        <i class="search icon"></i>
-    </div>
-    <div class="ui checkbox" onclick="{ filter.bind(this, undefined) }">
-        <label>Show Public Queues</label>
-        <input type="checkbox" ref="public">
+    <div class="ui horizontal list">
+        <div class="item">
+            <div class="ui icon input">
+                <input type="text" placeholder="Search by name..." ref="search"
+                       onkeyup="{filter.bind(this, undefined)}">
+                <i class="search icon"></i>
+            </div>
+        </div>
+
+        <div class="item">
+            <div class="ui checkbox" onclick="{ filter.bind(this, undefined) }">
+                <label>Show Public Queues</label>
+                <input type="checkbox" ref="public">
+            </div>
+        </div>
+        <div class="item">
+            <help_button href="https://github.com/codalab/competitions-v2/wiki/Competition-Creation:-Bundle" tooltip_position="right center"></help_button>
```

Comment:
to be consistent with the other `<help_buttons>` could probably put attributes on their own line?